CONTRIBUTING.md

```diff
@@ -1,12 +1,25 @@
 # Contributing to the documentation
 
-## Repository structure
+## Editing the documentation
+
+To edit the documentation:
+
+1. Learn about the general repository structure in the [Repository structure HTML](./README.html#repository-structure) or [Repository structure HTML](./README.md#repository-structure) section.
+
+2. Red the writing and markup conventions summary in the [Conventions and style guidelines](#conventions-and-style-guidelines)section.
+
+3. Ensure that you log an issue with the details of the problem in the [Documentation Repository Issue Tracker](https://github.com/fabric8io/fabric8-online-docs/issues) for context and assign it to yourself.
+
+4. Clone the documentation repository and make the relevant changes in a branch named after the issue you are working on.
+
+5. Locally build and test the changes. For details on the instructions and tools to do this, see [Locally building the documentation](#locally-building-the-documentation).
+
+6. After the local builds are successful, commit your changes and create a PR for the fixes. In the assignee field, select one of the writers to do a quick review before merging your changes. 
+
+7. For each push to the *master* branch and each PR, a preview build of the documentation is created and available at https://docs.prod-preview.openshift.io/. Build status: [![Build Status](https://ci.centos.org/view/Devtools/job/devtools-fabric8-online-docs-build-master/badge/icon)](https://ci.centos.org/view/Devtools/job/devtools-fabric8-online-docs-build-master/). The creator of the PR and reviewers can use this build for testing, which is why it is essential to test your changes locally before creating the PR.
```

Comment:
We want to have the build-status icon in the README for quick reference (it shows on the main page of the repo). It can be in the CONTRIBUTING, too, but it needs to be up front (I mean in the README).